### PR TITLE
LOG-1041: Changed 'Cluster Logging' in CSV to 'Red Hat OpenShift Logging'

### PR DESCRIPTION
**Pull request by vimalk78**
### Description
Changed 'Cluster Logging' in CSV to 'RedHat OpenShift Logging'

/cc @jcantrill 
/assign @jcantrill 

### Links
- JIRA: https://issues.redhat.com/browse/LOG-1041

### CODE REVIEW

File: manifests/5.0/cluster-logging.v5.0.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ metadata:
     categories: "OpenShift Optional, Logging & Tracing"
     certified: "false"
     description: |-
-      The Cluster Logging Operator for OCP provides a means for configuring and managing your aggregated logging stack.
+      The RedHat OpenShift Logging Operator for OCP provides a means for configuring and managing your aggregated logging stack.
```
Comment (jcantrill) [inline manifests/5.0/cluster-logging.v5.0.0.clusterserviceversion.yaml:20]: @vimalk78 i think this should be "Red Hat"  I'll update the issue.

Comment (vimalk78) [reply manifests/5.0/cluster-logging.v5.0.0.clusterserviceversion.yaml]: updated

**Comment by vimalk78** [comment]
waiting for https://github.com/openshift/origin-aggregated-logging/pull/2063 to merge

**Pull request merged**